DirectProgramming/DPC++FPGA/Tutorials/Features/ac_types/ac_int/src/ac_int.cpp

```diff
@@ -0,0 +1,350 @@
+//==============================================================
+// Copyright Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+// clang-format off
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/ext/intel/ac_types/ac_int.hpp>
+// clang-format on
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;
+using namespace std;
```

Comment:
Never. No.

Reply:
Boiler plate from some template. Will remove.

Reply:
Interestingly, other Features code samples need to be updated to remove such using directives as well.

Reply:
I am assuming your comment applies to `using namespace std;` and not to `using namespace sycl`
AFAIK the latter is allowed in code samples please let me know if that has changed.

Reply:
Updated.

Reply:
Yes.
`sycl` namespace is fine, not `std`